### PR TITLE
docs: suggest group policy update in desktop troubleshooting section

### PR DESCRIPTION
**Pull request by pschisa**
Update Disable NLA section to include troubleshooting step to force the policy update.  Similar to smartcard troubleshooting section and relevant here from customer experience

### CODE REVIEW

**Review by alexfornuto — APPROVED** [review]
Pushed a copy edit for your addition and something else I noticed. 👍

File: docs/pages/desktop-access/troubleshooting.mdx [hunk 1]
```diff
@@ -276,4 +276,11 @@ Teleport currently requires that NLA is disabled in order to perform its
 certificate-based passwordless login.
 
 To disable NLA, follow the instructions in our
-[Getting Started Guide](./manual-setup.mdx#allow-remote-rdp-connections).
+[Desktop Access Manual Setup](./manual-setup.mdx#allow-remote-rdp-connections).
+If you are still encountering this error after disabling NLA in Active Directory,
+run the following command from the Windows Desktop cmd prompt as an administrator
```
Comment (zmb3) [inline docs/pages/desktop-access/troubleshooting.mdx:281]: I would either say "command prompt" or "cmd.exe"

Comment (webvictim) [inline docs/pages/desktop-access/troubleshooting.mdx:281]:
```suggestion
run the following command from the Windows Desktop command prompt as an administrator
```

**Review by zmb3 — APPROVED** [review]

**Comment by alexfornuto** [comment]
Heads up, this may require manual backporting because of the changes coming to Desktop Access in v12.

**Pull request merged**